eventio/event_io_file.py

```diff
@@ -0,0 +1,65 @@
+import struct
+from .object_header import make_ObjectHeader
+from io import BytesIO
+
+class EventIOObject:
```

Comment:
Wait, so `self.header` is basically the path down the tree? Wow, that is really unexpected to me.